### PR TITLE
EPMLABSBRN-355

### PR DESCRIPTION
**Pull request by LisaTomaeva**
[EPMLABSBRN-355](https://jira.epam.com/jira/browse/EPMLABSBRN-355)

**Description**:

### CODE REVIEW

File: frontend/app/components/task-player/single-simple-words/component.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    set(task, 'completedInCurrentCycle', true);
+    set(task, 'nextAttempt', false);
+  }
+  markNextAttempt(task) {
+    set(task, 'nextAttempt', true);
+  }
+  startTask() {
+    this.isCorrect = false;
+  }
+  updateLocalTasks() {
```
Comment (lifeart) [inline frontend/app/components/task-player/single-simple-words/component.js:53]: if you extend component, we don't need this duplicated methods

File: frontend/app/components/task-player/single-simple-words/template.hbs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <AudioPlayer
+      @audioFileUrl={{this.audioFileUrl}}
+      @disabled={{@disableAudioPlayer}}
+    />
+    <ul class="task-player__options mt-5">
+      {{#each this.task.answerOptions as |answerOption|}}
+        <li class="task-player__option">
+          <button
+            data-test-task-answer
+            data-test-task-answer-option={{word}}
```
Comment (lifeart) [inline frontend/app/components/task-player/single-simple-words/template.hbs:13]: fix lint errros

File: frontend/app/models/task/single-simple-words.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import DS from 'ember-data';
+const { attr } = DS;
+import shuffleArray from 'brn/utils/shuffle-array';
+
+export default class TaskSingleSimpleWordsModel extends BaseTask.extend({
+  answerOptions: attr(),
+  exerciseType: 'single-simple-words',
+  tasksToSolve: computed('answerOptions.[]', function() {
+    return [
+      ...shuffleArray(this.answerOptions),
```
Comment (lifeart) [inline frontend/app/models/task/single-simple-words.js:12]: why we use double shuffle?

File: frontend/app/utils/task-item.js [hunk 4]
```diff
@@ -19,7 +19,7 @@ export class TaskItem {
       completedInCurrentCycle: this.completedInCurrentCycle,
       normalizedAnswerOptions: this.normalizedAnswerOptions,
       nextAttempt: this.nextAttempt,
-      answer: this.answer.slice(0),
+      answer: this.answer.length ? this.answer.slice(0) : this.answer,
```
Comment (lifeart) [inline frontend/app/utils/task-item.js:22]: answer can be non-array?

Comment (LisaTomaeva) [reply frontend/app/utils/task-item.js]: Fixed

File: frontend/app/components/task-player/single-simple-words/template.hbs (hunk 2 above)
Comment (LisaTomaeva) [reply frontend/app/components/task-player/single-simple-words/template.hbs]: Fixed

**Pull request merged**